WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerViewModel.kt

```diff
@@ -263,8 +263,15 @@ class MediaPickerViewModel @Inject constructor(
         } else if (domainModel?.isLoading == true) {
             PhotoListUiModel.Loading
         } else {
+            val stringId = when (if (domainModel?.mediaTypes?.size === 1) domainModel.mediaTypes.first() else null) {
```

Comment:
I would rather have the variable extracted out here.  I realize that Android Studio _helpfully_ points out that you can inline this variable if you do what I suggest though, so feel free to leave it as-is if you prefer.

```suggestion
            val onlyMediaType = if (domainModel?.mediaTypes?.size == 1) domainModel.mediaTypes.first() else null
            val stringId = when (onlyMediaType) {
```

Reply:
I'm with you on this one. Android studio kept bugging me to remove the extracted val. Thought it might cause linting issues if I ignored the suggestion. I'll change this.